dapps/marketplace/src/hoc/withGrowthCampaign.js

```diff
@@ -43,8 +43,10 @@ function withGrowthCampaign(WrappedComponent) {
                     // action is completed
                     fetchPolicy="network-only"
                   >
-                    {({ data, error }) => {
-                      if (error) {
+                    {({ data, error, networkStatus, loading }) => {
+                      if (networkStatus === 1 || loading) {
+                        return ''
+                      } else if (error) {
```

Comment:
Ideally for these higher order components we should try to avoid returning a different component depending on the network or loading state, unless there's a hard error.

Instead a few lines below I think we can do this:

`growthCampaigns={get(data, 'campaigns.nodes') || []}`

Reply:
Ah yes, probably better :)  Implemented your idea !